dnf-behave-tests/dnf/vars-parsing.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Correct parsing of dnf/vars directory
+ 
+ 
+@bz2091636
+Scenario: Using dnf with non-files in /etc/dnf/vars
+  Given I create directory "/{context.dnf.installroot}/etc/dnf/vars"
+    And I create directory "/{context.dnf.installroot}/tmp"
+  Given I successfully execute "ln -s {context.dnf.installroot}/tmp {context.dnf.installroot}/etc/dnf/vars/symlink"
+    And I use repository "dnf-ci-fedora"
+   When I execute dnf with args "repolist"
+   Then the exit code is 0
+    And stdout contains "repo id"
```

Comment:
It can also check `stderr` that it does not contains a traceback.

```
And stderr does not contain "Traceback"
```

Reply:
tbh. that's not a good check, we could have that check in every scenario. Checking the exit code should be enough. (I've always been wondering if checking `stderr` is empty in we don't have an explicit `stderr` check would be feasible. That'd be much more robust, but it's another discussion.)

Reply:
But it works - it is not sensitive to plugins, and wording of error message.

Reply:
@lukash I think that your proposal is good (to test every scenario stderr) but it requires another implementation that you proposed because there are several dnf commands that has not empty stderr. I also think that such a change shouldn't be included in this PR.

Reply:
I hope it's clear from my first comment I wasn't suggesting it for this PR...

Reply:
Sorry for the confusion